kolibri/plugins/learn/assets/src/views/SidePanelModal/SidePanelResourcesList.vue

```diff
@@ -16,21 +16,6 @@
     name: 'SidePanelResourcesList',
     $trs: {
       /* eslint-disable kolibri/vue-no-unused-translations */
-      noOtherLessonResources: {
```

Comment:
According to the comment in the template, we can delete this file entirely, so we should do so!